### ISSUE TITLE
Fish shell - TLDR "barfs" a bunch of text half way through command

### ISSUE THREAD

**Issue by cjappl** [tldr-pages/tldr#8310]
Steps to repro:

* Install tldr 
* Instantiate fish shell (seems to be the first time you use it only)
* start to type a command `tldr choo`  (trying to type `tldr choose`

Expected:
* Output doesn't come until you finish your command and press enter

Observed:
* See screenshot - tons of text outputted, maybe from fish autocomplete system?

Not sure if this is a fish bug or a tldr bug. This is the only program that has this issue 
<img width="1717" alt="Screen Shot 2022-08-05 at 1 51 36 PM" src="https://user-images.githubusercontent.com/14171107/183160990-d221be3e-bb70-4852-b31a-074406de781c.png">

installed tldr via homebrew.

tldr version: 1.4.3
fish version 3.5.1

**Comment by CleanMachine1** [comment]
How did you install your tldr client?

**Comment by mfrw** [comment]
@cjappl Could you please provide the steps to reproduce the issue:
- The client that you use for `tldr` 
- The version of `fish`
- Any other information you deem which might be helpful for us to isolate the issue :)

**Comment by cjappl** [comment]
Whoops, sorry, thought I responded! 

installed tldr via homebrew.

tldr version: 1.4.3
fish version 3.5.1

In terms of reproing, it seems to be the first time I try to use tldr in a shell session. I will update my original comment accordingly with this info.

**Comment by CleanMachine1** [comment]
Try updating the client, you are using our C client and the changelog for the next version includes

```
Fix fish completion not escaping characters
Make fish completion reflect actual usage of tldr better
```

**Comment by cjappl** [comment]
Cool! Thanks, just updated. So far that seems to work.

I'm going to close, and I'll re-open if I repro. Thanks for the help

For future reference the version working is: tldr v1.5.0 (v1.5.0)

**Comment by CleanMachine1** [comment]
Glad to help.

**Comment by cjappl** [comment]
Via homebrew
